src/components/Auth/FormInputGroup/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const FormInputGroup = ({ labelFor, label, inputType, inputId, forgotPassLink }) => {
+  return (
+    <div className="form-input-group">
+      <div className="form-input-group__container">
+        <label htmlFor={ labelFor } className="form-input-group__label">
+          { label }
+        </label>
+        <input
+          type={ inputType }
+          id={ inputId }
+          className="form-control form-input-group__box"
```

Comment:
@rafayet-monon What if we use `form-input-group__input` instead of `form-input-group__box`?
As the element isn't a box 🙌